admin/command_runner.go

```diff
@@ -266,7 +266,11 @@ func (r *CommandRunner) processLoop(ctx context.Context) {
 
 	for {
 		select {
-		case command := <-r.commandQ:
+		case command, ok := <-r.commandQ:
+			if !ok {
+				continue
```

Comment:
since the `commandQ` channel is only closed when shutting down, should this return instead? Otherwise, the loop will churn